server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -978,4 +978,38 @@ public void test_geo_shape_can_be_not_null() {
         ));
 
     }
+
+    @Test
+    public void test_add_geo_shape_column_keeps_geo_settings_and_adds_generated_expression() throws Exception {
+        execute("CREATE TABLE tbl (id INTEGER)");
+
+        execute("ALTER TABLE tbl ADD COLUMN g geo_shape " +
+            "GENERATED ALWAYS AS 'POLYGON (( 5 5, 30 5, 30 30, 5 30, 5 5 ))' " +
+            "INDEX using QUADTREE with (precision='123m', distance_error_pct='0.123')"
+        );
+
+        execute("insert into tbl (id) values (1)");
+        execute("refresh table tbl");
+        execute("select g from tbl where id = 1");
+
+        assertThat(
+            TestingHelpers.printedTable(response.rows()),
+            is("{coordinates=[[[5.0, 5.0], [5.0, 30.0], [30.0, 30.0], [30.0, 5.0], [5.0, 5.0]]], type=Polygon}\n")
+        );
```

Comment:
Not sure whether I need this assertion - probably `SHOW CREATE TABLE` is enough?

`SHOW CREATE TABLE` covers not only gen expression but also geo configs - I anyway need such test as I initially noticed this while working on `ADD COLUMN` (see https://github.com/crate/crate/pull/13238/commits/408ccd6524eb2a7ea0f666b76124d0c800bcda82) and this test covers that all configs are handled in planner and properly merged in a new transport action.

Reply:
Afaik the expression evaluation here does not differ from any other generated expression and such is already tested elsewhere. I think a show table test is enough for this case, but no strong opinion.

Reply:
I agree that we can simplify, removed it